src/translations/en.json

```diff
@@ -8,7 +8,21 @@
     "app.delete": "Delete",
     "app.details.label": "Details",
     "app.edit": "Edit",
+    "app.excel_upload.about.label": "This Form helps to upload data in the tool using excel.",
+    "app.excel_upload.added_file.label": "File Added!",
     "app.excel_upload.label": "Excel Upload",
+    "app.excel_upload.error_upload.header": "Error!",
+    "app.excel_upload.error_upload.label": "Could not apply changes",
+    "app.excel_upload.error.header": "Error when adding",
+    "app.excel_upload.error.label": "This document is not supported, please delete and upload another file.",
+    "app.excel_upload.instruction.label": "Drag & drop file or",
+    "app.excel_upload.instruction.choice": "choose file",
+    "app.excel_upload.selection.label": "Select a Table",
+    "app.excel_upload.success.header": "Success!",
+    "app.excel_upload.success.detail": "Your file has been successully uploaded!",
+    "app.excel_upload.upload_new_file.label": "Upload new file",
+    "app.excel_upload.upload_try_again.label": "Try again",
+    "app.excel_upload.view_template": "View Template",
```

Comment:
`"app..."` means it can be use in all app, instead of adding` "app.excel_uploud..."`  decide to use for example  `'app.error_upload.header" `or `"excel_upload.error_upload.header"` not both